lib/src/flutter_local_notifications.dart

```diff
@@ -217,6 +223,32 @@ class FlutterLocalNotificationsPlugin {
     });
   }
 
+  /// Shows a notification on the given day interval at the specified time
+  Future<void> showCustomDayIntervalAtTime(
```

Comment:
What's the reason behind having another method? If I'm not mistaken, it looks like an optional parameter for the interval in could've been added to `showDailyAtTime` method that would have a default value of zero for backwards compatibility. I'm guessing it was to avoid making changes to the existing method but IMO it's be cleaner to modify it and could potential do similar changes for the other methods.

Reply:
i actually like to have another method because it won't be daily - it will be in two or three days. the daily method could call this method with 1 though and weekly with 7 to make the code cleaner for future development

Reply:
Yep I understand it's not every day. What I'm suggesting here is identical to how recurrence rules are configured for calendar events (see https://tools.ietf.org/html/rfc5545#page-40 in case you're not familiar). So daily here is just the type of frequency

Reply:
Yeah I was thinking the same way. But I didn't want to adjust the existing methods like you've mentioned and I thought that the `showDailyAtTime` would become more of a shorthand to have daily notifications.
I'll update the PR with an optional parameter that will default to 1 day, thanks guys for your feedback!